documentation/source/newsfragments/1629.feature.rst

```diff
@@ -0,0 +1,2 @@
+Only ``Makefile.sim`` is required to be included in user Makefile (``Makefile.inc`` is not needed).
```

Comment:
```suggestion
Including ``Makefile.inc`` from user makefiles is now a no-op and deprecated. Lines like  ``include $(shell cocotb-config --makefiles)/Makefile.inc`` can be removed from user makefiles without loss in functionality.
```

Can you put this into the removals category of news fragments, not in the features one?